cmd/pilot-agent/main.go

```diff
@@ -221,6 +251,10 @@ func init() {
 		"IP Address and Port of a statsd UDP listener (e.g. 10.75.241.127:9125)")
 	proxyCmd.PersistentFlags().IntVar(&proxyAdminPort, "proxyAdminPort", int(values.ProxyAdminPort),
 		"Port on which Envoy should listen for administrative commands")
+	proxyCmd.PersistentFlags().StringVar(&controlPlaneAuthPolicy, "controlPlaneAuthPolicy",
+		values.ControlPlaneAuthPolicy.String(), "Control Plane Authentication Policy")
```

Comment:
Can we at least make the comment clear that NONE means no TLS/security for the control plane, and is strongly discouraged ?

Reply:
Also - make sure default is 'NONE' and add a comment that default will change to TLS (we need to allow upgrades)